vale/vocab.txt

```diff
@@ -0,0 +1,137 @@
+addr
+api
+apis
+architecting
+artifactory
+Artifactory
+Atlassian
+autogenerated
+autoscaling
+Autoscaling
+Basecamp
+Basscss
+Bazel
+Bitium
+boolean
+boomper
+Boomper
+Buildbox
+buildkite
+Buildkite
```

Comment:
Per the other change, we probably need `buildkit` in here.

Reply:
Heh yeah, I thought that was a typo and removed it, and snuck it back in in e7b7289